src/textual/eta.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+import bisect
+from math import ceil
+from time import monotonic
+
+import rich.repr
+
+
+@rich.repr.auto(angular=True)
+class ETA:
```

Comment:
Just because I'll never be able to live with myself if I *don't* mention it on record.... Textual no-TLA rule!

Reply:
This is an established abbreviation, where there is no doubt about how it is abbreviated.

There is precedence: "css" and not "cascading_style_sheet".